packages/compartment-mapper/src/import.js

```diff
@@ -44,13 +44,17 @@ export const loadLocation = async (read, moduleLocation) => {
       __shimTransforms__,
       Compartment
     });
-    return compartment.import(moduleSpecifier);
+    // Wrap import calls to bypass SES censoring for dynamic import.
+    // eslint-disable-next-line prettier/prettier
+    return (compartment.import)(moduleSpecifier);
   };
 
   return { import: execute };
 };
 
 export const importLocation = async (read, moduleLocation, options = {}) => {
   const application = await loadLocation(read, moduleLocation);
-  return application.import(options);
+  // Wrap import calls to bypass SES censoring for dynamic import.
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Added this boilerplate to stop `lint --fix` from erasing the “unnecessary” parentheses.